File: JAVA/JavaAssignment/src/main/Ques3.java
package main;

import java.util.Scanner;

public class Ques3 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("enter the string");
        String str=sc.nextLine();
        System.out.println("enter the charecter");
        char ch=sc.next().charAt(0);
        System.out.println("Length Of String:" + str.length());
        System.out.println("lenght of string without charecter"+str.replaceAll(String.valueOf(ch),"").length());
        int count= str.length()-str.replaceAll(String.valueOf(ch),"").length();
        System.out.println("the occurance of given chareceter "+ch+" is:"+count);
    }
}

File: KotlinAssignment1/src/Ques3.kt
import java.util.*
//Write a program to find the number of occurrences of a character in a string without using loop.
fun main(){
   val reader=Scanner(System.`in`)
    println("Enter the String")
    var mString=reader.nextLine()
    println("Enter the charecter")
    var mChar=reader.next()
    var totalLength=mString.length
    var replaceLength=mString.replace(mChar,"").length
    println("No of Occurance = ${totalLength-replaceLength}")


}